examples/renv-shiny/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM rhub/r-minimal:4.2.3-patched
+
+COPY installr /usr/local/bin/
+
+RUN wget https://github.com/jgm/pandoc/releases/download/2.13/pandoc-2.13-linux-amd64.tar.gz && \
+    tar xzf pandoc-2.13-linux-amd64.tar.gz && \
+    mv pandoc-2.13/bin/* /usr/local/bin/ && \
+    rm -rf pandoc-2.13*
+
+COPY . .
+
+RUN installr -d -e \
```

Comment:
This actually fails for me with 
```
10.93 Activating and Restoring renv.lock file if needed.
11.07 Error in loadNamespace(x) : there is no package called ‘renv’
11.07 Calls: loadNamespace -> withRestarts -> withOneRestart -> doWithOneRestart
11.07 Execution halted
```

Can you run `docker build .` locally?

Reply:
Tested with the new script and it worked.